ethcore/src/account.rs

```diff
@@ -120,7 +120,7 @@ impl Account {
 	}
 
 	/// Get (and cache) the contents of the trie's storage at `key`.
-	pub fn storage_at(&self, db: &AccountDB, key: &H256) -> H256 {
+	pub fn storage_at(&self, db: &HashDB, key: &H256) -> H256 {
```

Comment:
reason for this change? `AccountDB` provides a few more guarantees than just `HashDB` and it doesn't look like you've changed the call sites